index.js

```diff
@@ -29,6 +30,14 @@ function preziParser(url) {
   return match ? match[1] : url;
 }
 
+// TODO: Write regex for staging and local servers.
+const mfrRegex = /^http(?:s?):\/\/(?:www\.)?mfr\.osf\.io\/render\?url=http(?:s?):\/\/osf\.io\/([a-zA-Z0-9]{1,5})\/\?action=download/;
```

Comment:
I think this is overly restrictive. You should just check for something in the range of `https://osf.io/<guid>/` (or the subdomain variants `https://*.osf.io/<guid>/` and throw away the rest. That would make it easy to get the subdomain as well.